### PR TITLE
pt-br hockeyapp translation

### PR DESCRIPTION
**Pull request by Panthro**
The pt-br translation of hockey app strings.

I'm already using in my own apps, so I'm sharing with the community.

### CODE REVIEW

**Comment by ranterle** [comment]
Hi @Panthro, thanks for contributing a translation – this is awesome!
We will look into adding this to our set of localizations with the next release.

**Comment by Panthro** [comment]
@ranterle one of the things I had to do locally, was resize the UPDATE button to fit the word _"ATUALIZAR"_ which is bigger in portuguese.

I think it would be nice to externalize the values that are hardcoded in the views, like in the [activity update layout](https://github.com/bitstadium/HockeySDK-Android/blob/develop/hockeysdk/src/main/res/layout/hockeyapp_activity_update.xml#L59) so they can be overwriten in other languages to fit the content.

What do you think?

**Comment by Panthro** [comment]
@ranterle any update on this?

I'm already using locally, but have new projects every now and then and would like the sdk to come with the translation I most use by default.

Thanks

**Comment by Panthro** [comment]
wow guys, over 5 months for a translations PR?

**Comment by ranterle** [comment]
@Panthro Sorry for the delay - we are looking for the right people to verify the translation. While a one-time PR is great and we really appreciate your effort, we also have to make sure we are able to support the language in the future, if we change things about the UI.
That said, I'm positive we will be able to merge this in our upcoming release soon. Please stay tuned.

File: hockeysdk/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <string name="hockeyapp_feedback_email_hint">Email</string>
+    <string name="hockeyapp_feedback_subject_hint">Assunto</string>
+    <string name="hockeyapp_feedback_message_hint">Mensagem</string>
+    <string name="hockeyapp_feedback_last_updated_text">Ultima atualização: %s</string>
+    <string name="hockeyapp_feedback_attachment_button_text">Adicionar anexo</string>
+    <string name="hockeyapp_feedback_send_button_text">Enviar feedback</string>
+    <string name="hockeyapp_feedback_response_button_text">Adicionar Resposta</string>
+    <string name="hockeyapp_feedback_refresh_button_text">Atualizar</string>
+    <string name="hockeyapp_feedback_title">Feedback</string>
+    <string name="hockeyapp_feedback_send_generic_error">A mensagem não pôde ser enviada. Por favor verifique os campos, sua conexão com a internet e tente novamente..</string>
```
Comment (ranterle) [inline hockeysdk/src/main/res/values-pt/strings.xml]: …

**Comment by Panthro** [comment]
amazing!! I'll try the new version when available and remove my own custom changes, I'll give feedback if anything does not work as expected.

Thanks!

**Comment by ranterle** [comment]
Sorry for the long wait on this - we cleared up responsibilities around localizations and also finished other tasks which blocked us 😉  - thanks for staying with us!

**Comment by Panthro** [comment]
It would be nice if you guys could take a look at this as well:

https://github.com/bitstadium/HockeySDK-Android/pull/159#issuecomment-223029679

thanks :)

**Comment by ranterle** [comment]
c6ff80b334f40af7ae2f56b60a127b026734444d should take care of that.

**Pull request merged**